doc/release-notes/release-notes-5.2.0-rc1.md

```diff
@@ -0,0 +1,200 @@
+Notable changes
+===============
+
+Node Performance Improvements
+-----------------------------
+
+This release makes several changes to improve the performance of node operations.
+These include:
+
+- Backported CuckooCache from upstream to improve the performance of signature
+  caching.
+
+- Added caching of proof and signature validation results for Sapling and
+  Orchard to eliminate redundant computation.
+
+- Backported SHA-256 assembly optimizations from upstream.
+
+Wallet Performance Improvements
+-------------------------------
+
+This release makes several changes to improve the performance of wallet operations.
+These include:
+
+- We now parallelize and batch trial decryption of Sapling outputs.
+
+- We now prune witness data in the wallet for notes spent more than 100 blocks
+  in the past, so that we can avoid unnecessarily updating those witnesses.
+  In order to take advantage of this performance improvement, users will need
+  to start their nodes with `-rescan` the first time .
```

Comment:
A little more clear would be:

> `-rescan` once to reset the wallet

Reply:
I had been typing a more complete comment got interrupted in the middle and didn't complete it. Hence the space before the period. I'll fix it before the release.